build/cloudbuild-master.yaml

```diff
@@ -24,9 +24,13 @@ steps:
   args: ['build', '--target', 'pre-build', '-t', 'gcr.io/$PROJECT_ID/forseti-prebuild:master', '.']
 - name: 'gcr.io/cloud-builders/docker'
   id: Build forseti-security master container image
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/forseti:master', '.']
+  args: ['build',  '--target', 'runtime', '-t', 'gcr.io/$PROJECT_ID/forseti:master', '.']
+- name: 'gcr.io/cloud-builders/docker'
+  id: Build forseti-server master container image
+  args: ['build', '--target', 'forseti-server', '-t', 'gcr.io/$PROJECT_ID/forseti-server:master', '.']
 
 images:
+- gcr.io/$PROJECT_ID/forseti-server:master
 - gcr.io/$PROJECT_ID/forseti:master
```

Comment:
Will the forseti image refer to the client only now? If so, it might be better to rename this image as forseti-client.

Reply:
@gkowalski-google I'd like to hold off on renaming it here.  I don't think I'll be able to update the helm-charts in time for the next release to include the `forseti-server` container image anyway.  This way, the helm-charts and Terraform module will still be able to use the image named `forseti`.

Reply:
OK makes sense, should we create a ticket to document what else needs to be completed for this change?

Reply:
@gkowalski-google I agree.  I'll do that now.

Reply:
https://github.com/forseti-security/forseti-security/issues/3511